lib.coffee

```diff
@@ -18,9 +18,12 @@ class DeflatePartStream extends DeflateCRC32Stream
 		super
 	push: (chunk) ->
 		if chunk isnt null
-			# got another chunk, previous chunk is safe to send
-			super(@buf)
-			@buf = chunk
+			if chunk.length >= 2
```

Comment:
@zvin this is "2" because `DEFLATE_END` is 2-byte long? Would it be possiblke to mention this in the commit message? Otherwise it might be easy to miss when any maintenance?

Or even better, if possible, I've seen the pattern somewhere to have something "DEFLATE_END_LENGTH = 2"  constant, or so, and all where there are the relevant "2" values, to use that constant which would explain the use naturally? Like `chunk.length >= DEFLATE_END_LENGTH` and `@buf.length >= DEFLATE_END_LENGTH and @buf[-1*DEFLATE_END_LENGTH..].equals(DEFLATE_END)` or something similar...

Does it make any sense?

This is good as it is for me, but this above would make things *nicer* maybe?

Reply:
Good idea @imrehg , updated.
We could probably also remove the `buf[-1*DEFLATE_END_LENGTH..].equals(DEFLATE_END)` check as `DeflateCRC32Stream` (which inherits `zlib.DeflateRaw`) will always end streams with these 2 bytes. Leaving as it is for now as the check is cheap.

Reply:
Cool, also good to know that `-VARIABLENAME` works as well, basic little TIL. :)
Looks great for now!